samcli/lib/utils/sam_logging.py

```diff
@@ -28,9 +38,19 @@ def configure_logger(logger, formatter, level):
         if handlers:
             log_stream_handler = handlers[0]
         else:
-            log_stream_handler = logging.StreamHandler()
+            log_stream_handler = (
+                RichHandler(console=Console(stderr=True), show_time=False, show_path=False, show_level=False)
+                if sys.stderr.isatty()
+                and not any(
+                    [
+                        os.getenv(LOGGING_COLOR_ENV_VAR),
+                        os.getenv(SAM_LOGGING_COLOR_ENV_VAR),
```

Comment:
Should this environment variables called `NO_LOGGING..` and `SAM_NO_LOGGING..` to make it more readable?

Reply:
Done